go/store/nbs/store.go

```diff
@@ -214,6 +224,25 @@ func (nbs *NomsBlockStore) GetChunkLocations(ctx context.Context, hashes hash.Ha
 
 		return ranges, nil
 	}
+
+}
+
+func (nbs *NomsBlockStore) GetChunkLocations(ctx context.Context, hashes hash.HashSet) (map[string]map[hash.Hash]Range, error) {
```

Comment:
Seems like callers need to migrate to the Paths method if they want to resolve to an actual file. Changing this here looks weird to me.